node_helper.js

```diff
@@ -1,5 +1,5 @@
 const Log = require("logger");
-const fetch = require("node-fetch");
```

Comment:
You didn't replaced `node-fetch` by `fetch`. You replaced `node-fetch` by deprecated `request`...

Reply:
fetch cannot be used with require.

Feel free to open an MR with an alternative approach.

Reply:
Sure. Look at #32 :slightly_smiling_face: